hazelcast-full.yaml

```diff
@@ -181,14 +181,14 @@ spec:
   rbac:
     # Specifies whether RBAC resources should be created
     # It is not required if DNS Lookup is used (https://github.com/hazelcast/hazelcast-kubernetes#dns-lookup)
-    create: true
+    create: false
 
   serviceAccount:
     # Specifies whether a ServiceAccount should be created
-    create: true
+    create: false
```

Comment:
this is true by default in helm chart.

Reply:
Why are we changing it?

Reply:
I see. That is the reason introduced with that change.`name: hazelcast` 

We don't want to autogenerate SA because it is causing some problem?

Reply:
We can have it autogenerated. Then `operator-rbac.yaml` would have more requirements and the user would need to change one of its fields: `namespace`, because it would differ for each namespace.

Not sure what's better, I find these instructions as kind-of code sample, so probably less instructions is better (what is here right now). But to be honest, I don't have a strong opinion on that, I can change it to autogenerated. It would mean removing `hazelcast-rbac.yaml` , changing `operator-rbac.yaml` and adding more instructions to README.

Reply:
less is better. Changing names of those yamls to sample-hazelcast-full.yaml etc. could be an option too. Or some comment on the top of the file.

We need to make it easier for people to try.